package.json

```diff
@@ -54,7 +54,7 @@
   "lint-staged": {
     "*.{js,ts,svelte,html,md,svx}": "eslint --cache --fix",
     "*.{css,scss,svelte,html,js,ts,svx,md}": [
-      "prettier --cache --write"
+      "prettier --cache --ignore-unknown --write"
     ]
```

Comment:
Should we expand this to json, json5, yml, mjs, etc? Or maybe a catch all `*`

Reply:
thanks, * is correct together with --ignore-unknown.